docker-compose.yml

```diff
@@ -1,15 +1,18 @@
 services:
   postgres:
     image: postgres:16
+    container_name: hostedgpt_postgres
     restart: always
     environment:
-      POSTGRES_USER: app
-      POSTGRES_DB: app_development
-      POSTGRES_PASSWORD: secret
+      # NOTE: If you use DATABASE_URL, it needs to match these env vars
+      #  so you can either have it match the default values for these env vars
+      #  or you can set these env vars to match the user and password from the url
```

Comment:
this is the issue with using a url and running with docker. If you end up changing the username or password then you have to change it in two places. Pretty minor issue, but still not ideal

Reply:
Oops, I asked about this in a different thread so I’ll resolve this one